ui/src/app/base/components/ArrowPagination/ArrowPagination.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, Icon } from "@canonical/react-components";
+
+type Props = {
+  currentPage: number;
+  itemCount: number;
+  pageSize: number;
+  setCurrentPage: (page: number) => void;
+};
+
+const ArrowPagination = ({
+  currentPage,
+  itemCount,
+  pageSize,
+  setCurrentPage,
+}: Props): JSX.Element => {
+  const onFirstPage = currentPage === 1;
+  const onLastPage =
+    itemCount === 0 || currentPage === Math.ceil(itemCount / pageSize);
+
+  return (
+    <>
+      <Button
+        appearance="base"
+        className="u-no-margin--right u-no-margin--bottom"
+        disabled={onFirstPage}
+        onClick={() => {
```

Comment:
Is it possible to add an `aria-label` to these buttons?

Reply:
Yep, will do.